From fd3e9815846aa1efe295e897faf1e57f38c6c165 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 7 Jun 2023 14:54:34 +0200
Subject: [PATCH] man: suffix --unit with an equal sign, since it expects an
 argument

As per our usual syntax in the docs.

(cherry picked from commit e754af353c494edfdd25412fae32876a2772a5cd)

Related: RHEL-31070
---
 man/journalctl.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/man/journalctl.xml b/man/journalctl.xml
index 5bf895fce4..29d06aaef8 100644
--- a/man/journalctl.xml
+++ b/man/journalctl.xml
@@ -110,8 +110,8 @@
         <option>--user</option>).  If neither is specified, show all messages that the user can see.
         </para>
 
-        <para>The <option>--user</option> option affects how <option>--unit</option> arguments are
-        treated. See <option>--unit</option>.</para></listitem>
+        <para>The <option>--user</option> option affects how <option>--unit=</option> arguments are
+        treated. See <option>--unit=</option>.</para></listitem>
       </varlistentry>
 
       <varlistentry>
@@ -285,8 +285,8 @@
         <citerefentry><refentrytitle>systemd.slice</refentrytitle><manvolnum>5</manvolnum></citerefentry>
         unit, all logs of children of the slice will be shown.</para>
 
-        <para>With <option>--user</option>, all <option>--unit</option> arguments will be converted to match
-        user messages as if specified with <option>--user-unit</option>.</para>
+        <para>With <option>--user</option>, all <option>--unit=</option> arguments will be converted to match
+        user messages as if specified with <option>--user-unit=</option>.</para>
 
         <para>This parameter can be specified multiple times.</para></listitem>
       </varlistentry>
